DataManagementSystem/Client/DataManager.py

```diff
@@ -118,7 +118,7 @@ def __hasAccess( self, opType, path ):
     else:
       paths = path
 
-    res = self.fc.hasAccess( opType, paths )
+    res = self.fc.hasAccess( paths, opType )
```

Comment:
No, this is a change to make the calling signature "standard" where the LFNs argument is the first one